### DISCUSSION TITLE
Google Play Deep link Issues

### DISCUSSION THREAD

**Discussion by jackmorganxyz** [buildnatively/community#72] [Ideas]
### Please confirm you've checked the following resources:

- [X] I've searched through the [Docs](https://docs.buildnatively.com).
- [X] I've checked [Natively Youtube Playlist 1](https://youtube.com/playlist?list=PLYo9EJqlVMNdtzb_rm6_6kjlDCmaoAqlO&si=sOygZtRDGUn_2Yi8) and [Natively Youtube Playlist 2](https://youtube.com/playlist?list=PLYo9EJqlVMNcwz2o1Xx5xVYYzzKgcyMeB&si=f4GuCPvUAdjvHplD).
- [X] I've reviewed the [Troubleshooting Guides](https://docs.buildnatively.com/guides/troubleshooting).
- [X] I've visited the [FAQ](https://docs.buildnatively.com/natively-platform/faq).

### Which web app platform are you using?

Bubble

### Which SDK are you using?

Bubble Plugin

### JS SDK / Bubble Plugin Version (e.g., 2.12.1)

2.12.7

### Platform with the issue?

Android

### App ID

1687887203008x427975791387607040

### Steps to reproduce

It seems like the links Google is trying to access is incorrect. How do I update the links to my apps url 'app.petgenius.ai'?

Also note that in the top right corner the code says 'com.base.app.Core.JasonViewActivity', is this supposed to be json view activity? If this is pulling from the AndroidManifest.xml which should be present somewhere I don't have access to to edit this.

![Screen Shot 2024-01-29 at 10 48 51 AM](https://github.com/buildnatively/community/assets/66273764/d4756a0b-c983-47ad-8683-b0370239b9f5)

![Screen Shot 2024-01-29 at 10 58 14 AM](https://github.com/buildnatively/community/assets/66273764/dcb59e00-1a82-4ed2-876a-e24b7d48c3ac)

### Expected results

No having these issues present in the Google play dashboard.

### Actual results

No deep linking is working because the links are incorrect in the Google play dashboard, which seems like its being pulled from somewhere as I cant edit them.

**Comment by jackmorganxyz** [comment]
@romanfurman6 Can you please help me?

**Reply by romanfurman6** [reply]
@jackpmorgan checking ..

**Reply by romanfurman6** [reply]
@jackpmorgan can you please rebuild the app. It looks like an old issue, we've fixed it already

**Reply by romanfurman6** [reply]
Hello @jackpmorgan did you have a chance to check if that fix works for you?
